learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -15,70 +15,91 @@ redirect_from:
 
-Let’s start implementing a Ballerina service and client for the above definition. Create a new service definition file (i.e., `helloworld.proto`) in which you want to create the project and copy the above service definition.
+  package helloworld;
 
-## Implementation
+  // The greeting service definition.
+  service Greeter {
+  // Sends a greeting.
+  rpc sayHello (HelloRequest) returns (HelloReply);
+  }
 
-As illustrated in the diagram below, you first need to compile the service definition file (i.e., `helloworld.proto`), and generate the source code for both the client and service applications. In Ballerina, you can generate the source code using the built-in `Protocol Buffers to Ballerina` tool.
+  // The request message with the user's name.
+  message HelloRequest {
+  string name = 1;
+  }
 
-![gRPC Getting Started](/learn/images/grpc-getting-started.png)
+  // The response message with the greetings.
+  message HelloReply {
+  string message = 1;
+  }
+  ```
 
-Let’s walk through the instructions on implementing the gRPC service and client in the Ballerina language.
+Now, let’s implement the gRPC service and client in the Ballerina language.
 
-### Developing a gRPC service
+## Implementing the gRPC service
 
 As with any other Ballerina Program, you need to create a Ballerina project. Then, you can generate the service code in the project and write the business logic.
 
-#### Creating a Ballerina project
+### Creating the service project
 
-Execute the following command to create a new Ballerina project in the directory that you want to create the project.
+In the terminal, navigate to the `grpc` directory, and execute the command below to create the Ballerina project for the gRPC service implementation:
+
+> **Note:** For more information on Ballerina packages, see [Organizing Ballerina code](/learn/organizing-ballerina-code/).
 
 ```bash
 $ bal new greeter_service
 ```
 
-> **Note:** For more information on creating Ballerina packages, see [Getting Started with Ballerina](/learn/getting-started-with-ballerina).
+You view the output below.
+
+```bash
+Created new package 'greeter_service' at greeter_service.
+```
+
+This creates a directory named `greeter_service` with the files below.
 
-Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
+```bash
+.
+├── Ballerina.toml
+└── main.bal
+```
 
-#### Generating the source code
+>**Tip:** Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
 
-Execute the following command inside the same directory in which you created the Ballerina project to generate the source code related to the service definition.
+### Generating the source code of the service
+
+In the terminal, from inside the same `grpc` directory, execute the command below to generate the source code related to the service definition.
 
 ```bash
 $ bal grpc --mode service --input helloworld.proto --output greeter_service/
```

Comment:
How can we avoid getting `$` copied when copying the command. For ex: see [1]. It doesn't copy the `$` when copying.

[1] https://developers.google.com/style/code-syntax?hl=en

Reply:
Created https://github.com/ballerina-platform/ballerina-dev-website/issues/3962 to track this.

Reply:
Will be addressed via https://github.com/ballerina-platform/ballerina-dev-website/issues/3962.